pkg/tfvars/azure/azure.go

```diff
@@ -135,6 +135,7 @@ func TFVars(sources TFVarsSources) ([]byte, error) {
 		VMNetworkingType:                masterConfig.AcceleratedNetworking,
 		RandomStringPrefix:              randomStringPrefixFunction(),
 		VMArchitecture:                  vmarch,
+		ExtraTags:                       masterConfig.Tags,
```

Comment:
I see that the machines have the tags passed to them, but is this `ExtraTags` field used yet? Some of the providers have the installconfig tags passed to the tfvars initiated in `assets/cluster/tfvars.go`, should they be added there too or did I miss those tags?

Reply:
Thank you for reviewing @barbacbd!

> I see that the machines have the tags passed to them, but is this ExtraTags field used yet?

Not until now, but with this change ExtraTags will be used for adding user-defined tags to azure resources created by installer.

ExtraTags is part of the azure platform specific config which is made available in the platform specific terraform variable file with the name `azure_extra_tags`. `Generate()` method in `assets/cluster/tfvars.go` generates the platform terraform variable file by calling `azure.TFVars()` method. azure_extra_tags is used in terraform code for applying tags it's initialized with to the azure resources created.